### DISCUSSION TITLE
Cannot find module '@aws-sdk/client-s3

### DISCUSSION THREAD

**Discussion by aligajani** [stefanzweifel/sidecar-browsershot#129] [Q&A]
```
Hammerstone\Sidecar\Exceptions\LambdaExecutionException: Lambda Execution Exception for Wnx\SidecarBrowsershot\Functions\BrowsershotFunction: "Error: Cannot find module '@aws-sdk/client-s3'
Require stack:
- /var/task/browsershot.js
- ```

**Comment by stefanzweifel** [comment]
Can you tell us which package version you're using and Node version your function is using? 
If you're on v1, please upgrade to v2.

A couple of issues popped up the last couple of days which all could be resolved by updating the package: #127, #128.
